### PR TITLE
cd, find, ipconfig, msiexec, nvm, set, where: add Russian translation

### PR DESCRIPTION
**Pull request by iridacea**
Russian translation for several Windows utils.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by iridacea** [comment]
Can anyone help me to remove the last merge commit (named "Merge branch 'tldr-pages:main' into win")? Somehow it sneaked into my repo on Github, but there's no such commit in my local branch. :confused: 

This is the state right after `git pull --all` on the `main` branch:

```
*   43bdb2afb 2022-10-15 | Merge branch 'tldr-pages:main' into win (origin/win) [Irina]
|\
| * d25ee7fce 2022-10-14 | git-am: add Korean translation (#8955) [Jessica (Ji Sun) Lee]
| * 3b3ee1256 2022-10-14 | cmd, curl, exit, ftp: add German translation (#8776) [Lena]
* | 12f4e8451 2022-10-14 | where: add Russian translation (win) [Irina]
* | c15d24691 2022-10-14 | cd: add Russian translation [Irina]
* | 086cd7b8b 2022-10-14 | find: add Russian translation [Irina]
* | ece72870d 2022-10-14 | msiexec: add Russian translation [Irina]
* | eeeb63ee5 2022-10-14 | nvm: add Russian translation [Irina]
* | 0a3eaa92b 2022-10-14 | ipconfig: add Russian translation [Irina]
* | 5d86f0e12 2022-10-14 | set: add Russian translation [Irina]
|/
| * 96b2c13f8 2022-10-14 | find: double qoutes are mandatory for find on Windows (origin/find-win, find-win) [rozie]
|/
| * 413546ab7 2022-10-15 | where: the `/q` parameter doesn't display anything (origin/where, where) [Irina]
|/
* 325a98f99 2022-10-14 | apt, wajig: add Polish translation (#9020) (HEAD -> main, origin/main, origin/HEAD) [rozie]
* c5d0bcfa2 2022-10-14 | reboot: add pt_BR translation (#9014) [Felix Stenian]
* 1c0cbb486 2022-10-14 | xcopy: add German translation (#9009) [Julian Hüppauff]
```

**Comment by kbdharun** [comment]
I don't think you need to remove the merge/sync with the main since it has already occurred. Our main branch is isolated from other branches. And when we merge this PR only the modified files shown in `Files changed` will be added. I will review this PR now.

**Comment by iridacea** [comment]
I solved my problem with the extra merge commit. It looks like I've clicked the `Sync fork` button, while the `win` branch was selected, not the `main`. And that led to merging `tldr-pages:main` into my `iridacea:win`, which I didn't mean to.

I sovled the issue by force-pushing my `win` branch to previous commit. For the state described above it is:

```sh
git checkout win
git push --force origin win
```

I think you are right, @kbdharun, I didn't really need to revert that commit, but I decided to make it for purity. Anyway all these manipulations didn't affect the files in the `Files changed` section, I checked that.

So now this PR is finally ready for review. @kbdharun @EmilySeville7cfg

**Comment by iridacea** [comment]
@marchersimon Could you take a look at this?

**Review by marchersimon — APPROVED** [review]
I can't review the contents, but the layout looks fine.
Have you checked that your translations are the same as in [common-arguments.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)?

File: pages.ru/windows/msiexec.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Установить MSI-пакет с веб-сайта:
+
+`msiexec /package {{https://example.com/installer.msi}}`
+
+- Установить MSP-пакет с обновлением (патчем):
+
+`msiexec /update {{путь/до/файла.msp}}`
+
+- Удалить программу или обновление, используя соответствующий пакет MSI или MSP :
```
Comment (navarroaxel) [inline pages.ru/windows/msiexec.md:18]:
```suggestion
- Удалить программу или обновление, используя соответствующий пакет MSI или MSP:
```

**Review by navarroaxel — APPROVED** [review]

**Comment by iridacea** [comment]
> I can't review the contents, but the layout looks fine. Have you checked that your translations are the same as in [common-arguments.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)?

Yes, you're right. I actually left one token untranslated in `where.md`. I'll make a PR for that.

And as for `cd.md`, I used the form with backslashes like in English version of the page (`{{path\to\directory}}` → `{{путь\до\каталога}}`) . So since it's already "nonstandard", I decided to user the synonym `каталог`, not `папка`, to match with the word in the descriptions. What do you think? Should I change it from `каталог` to `папка` to be consistent with `common-arguments.md`?

**Pull request merged**